src/collections/teams.coffee

```
@@ -46,18 +46,30 @@ exports.deleteTeam = (team, callback) ->
 
 # Loads all items associated with a team, optionally limited by the types array
 exports.bulkLoad = (teamId, types, callback) ->
-  unless @isId teamId
-    throw new TSArgsError 'teamsnap.bulkLoad', 'teamId must be provided'
+  if typeof teamId is 'object'
+    loadParams = teamId
+    # Using params object (smartload)
+    unless @isId loadParams.teamId
```

Types are check down bellow and will default to null, I assume if no types it will return no items, but it will have commands and the other info apiv3 gives you. I don't think it should throw an error if no types are given since the info it would provide is still useful while developing.